docs/frontend/liquid-tags.md

```diff
@@ -81,6 +81,6 @@ Liquid Tags should also be accompanied by tests in `spec/liquid_tags` which conf
 
 Some Liquid Tags are constructed using HTML and CSS within the app, and some are constructed by displaying an iframe of an external site.
 
-CSS for Liquid Tags are found in `app/assets/stylesheets/ltags`. Liquid tag classes should generally be prepnded by `ltag__`. e.g. `ltag__tag__content` etc.
+CSS for Liquid Tags are found in `app/assets/stylesheets/ltags`. Liquid tag classes should generally be pretended by `ltag__`. e.g. `ltag__tag__content` etc.
```

Comment:
Looks like a "t" snuck in there somehow. `prepended` would be the one one in this case.

Reply:
Ohh there I tried to fix the typo with another type 😅. Now it should be `prepended`.